mlcomp/parallelm/components/restful/uwsgi_broker.py

```diff
@@ -125,6 +126,19 @@ def _generate_ini_file(self, shared_conf, entry_point_conf):
 
         return ini_filepath
 
+    @staticmethod
+    def _include_egg(filename):
+        if filename.endswith(".egg"):
+            py_ver_sub_str = re.search('-py.', filename)
```

Comment:
what if filename is:
`lala-python-py3.egg`
Probably not the case, but just generalization.

Reply:
You could save  one nested if with this:    
       if filename.endswith(".egg"):
            py_ver_sub_str = re.search('-py.', filename)
            if py_ver_sub_str and str(sys.version_info[0]) not in py_ver_sub_str.group(0):
                    return False
            return True
        return False

Reply:
As for your first comment - the existing code covers it.
As for you suggestion - what would happen if the file name is 'something-py2.py3.egg`.

I'll pick something more robust.